spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/index.adoc

```diff
@@ -17,6 +17,7 @@ Eric Bottard, Florian Rosenberg, Sabby Anandan, Marius Bogoevici, Mark Fisher, I
 :scst-starters-core-version: 1.0.4.RELEASE
 :sct-starters-core-version: 1.0.1.RELEASE
 
+:spring-cloud-stream-docs: http://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/index.html
```

Comment:
Ahh, found it. Also found a "bad" link for RxJava - https://github.com/spring-cloud/spring-cloud-dataflow/pull/997

We do have the SCSt version we use - so I'll add that on merge